blog/Webots-8-6-0-release.md

```diff
@@ -0,0 +1,134 @@
+# Version 8.6.0 Released
+
+<p id="publish-data">By Tom Norton - 14th July 2017</p>
+
+---
+
+Today we're happy to announce the release of Webots 8.6.0. This new release brings a ton of new features and improvements, coupled with the same dedicated patches addressing bugs and regressions. Listed below are some of the key features of this release (for a full list of changes please refer to the changelog (insert link here)).
+
```

Comment:
Sounds good to me!

Is the changelog link always https://www.cyberbotics.com/dvd/common/doc/webots/ChangeLog.html ?